src/complate/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"dependencies": {
-		"rodunj": "git+https://github.com/complate/rodunj.git",
+		"complate-ast": "https://github.com/larsrh/complate-ast/releases/download/v0.0.2/complate-ast-0.0.2.tgz",
```

Comment:
0.0.2 is not out yet (I thought I'd wait with the release until I get confirmation the whitespaces are good)